packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -172,13 +173,14 @@ export class GraphQLTransform {
    * @param schema The model schema.
    * @param references Any cloudformation references.
```

Comment:
Nit: This line in comment can be changed to the correct description of the newly added param